R/conclass.R

```diff
@@ -76,10 +89,42 @@ Conos <- R6::R6Class("Conos", lock_objects=FALSE,
       self$samples <- c(self$samples, x);
     },
 
+    #' @description Build the joint graph that encompasses all the samples, establishing weighted inter-sample cell-to-cell links
+    #'
+    #' @param k (default=15)
+    #' @param k k.self (default=10)
```

Comment:
There are lots of redundant `k` here :)

Reply:
Uff! It's embarrassing I never noticed this :) Quick copy/pasting error. 

Fixed now.